packages/flutter/lib/src/rendering/shifted_box.dart

```diff
@@ -262,6 +249,49 @@ class RenderPositionedBox extends RenderShiftedBox {
     markNeedsLayout();
   }
 
+  /// Subclasses must override this method to perform the actual layout of the
+  /// child. If there is a child, then the subclass should call
+  /// `super.performLayout()` to have this class perform the actual alignment.
+  /// If there is no child, do not call `super.performLayout()`.
+  ///
+  /// For more details, see [RenderShiftedBox.performLayout].
+  @override
+  void performLayout() {
```

Comment:
I'd probably cal this function something other than performLayout because it's only doing part of the contract.